test/unit/src/common/actions/t_repositories.js

```diff
@@ -132,7 +132,7 @@ describe('repositories actions', () => {
       it('should store no pageLinks', async () => {
         await store.dispatch(fetchUserRepositories());
         expect(store.getActions()).notToHaveActionOfType(
-          ActionTypes.SET_REPOSITORY_PAGE_LINKS
+          ActionTypes.SET_REPO_PAGE_LINKS
```

Comment:
It probably doesn't exist, because I think I only seen it in this one place in test when renaming.